tracer/src/Datadog.Trace/IAST/IastModule.cs

```diff
@@ -29,7 +29,7 @@ public IastModule()
             return null;
         }
 
-        return GetScope(Tracer.Instance, algorithm, integrationId, VulnerabilityType.WeakCipher, OperationNameWeakCipher, iast);
+        return GetScope(Tracer.Instance, algorithm, integrationId, VulnerabilityType.WEAK_CIPHER.ToString(), OperationNameWeakCipher, iast);
```

Comment:
For performance reasons, to avoid reflection, using strings instead of enums was suggested:
https://github.com/DataDog/dd-trace-dotnet/pull/3225#discussion_r989174808
https://github.com/DataDog/dd-trace-dotnet/pull/3225#discussion_r989167583

Reply:
Fixed in ae5e92bc5389e20ed3aec1e5c83eae04e7b9bed0